gulpfile.js

```diff
@@ -38,7 +40,7 @@ gulp.task('coveralls', function () {
 gulp.task('testCI', ['lint', 'style', 'cover'], function () {
   if (process.env.NODE_ENV !== 'test') {
     gulp.src(process.env.COVERAGE_DIR + '/coverage')
-      .pipe(plugins.clean());
+      .pipe(vinylPaths(del));
```

Comment:
@leore There was some discussion of this with @pon in the initial PR: https://github.com/lob/lob-node/pull/114